scss/layout/_aside.scss

```diff
@@ -2,7 +2,7 @@
 .n-content-layout {
 
 	&[data-layout-name="aside"] {
-		background-color: rgba(oColorsGetPaletteColor('wheat'), 0.5);
+		background-color: oColorsGetTint('wheat', 100);
```

Comment:
is the tint doing anything here?

Reply:
yep. tint works in an unexpected way - for a given colour hue and saturation are fixed I think, but tint sets the luminosity, or something like that. 

wheat's default luminosity is this

![image](https://user-images.githubusercontent.com/447559/34991114-5653944a-fac0-11e7-908b-56fda8cc4b34.png)

Reply:
ah interesting!